chia/_tests/wallet/sync/test_wallet_sync.py

```diff
@@ -94,7 +96,6 @@ async def test_request_block_headers(
     assert len(bh) == 6
     assert [x.reward_chain_block.height for x in default_400_blocks[10:16]] == [x.reward_chain_block.height for x in bh]
     assert [x.foliage for x in default_400_blocks[10:16]] == [x.foliage for x in bh]
-    assert [x.transactions_filter for x in bh] == [b"\x00"] * 6
```

Comment:
Considering we get two transaction blocks in this list, it's wrong to assume that all transactions filters are empty here, because reward coins at the minimum should be considered for those transaction blocks.

Reply:
the reason these fields used to be 0 is because of the bug, right?

Reply:
Yes.